### PR TITLE
Meson port

### PR DESCRIPTION
**Pull request by Salamandar**
Well I guess i'm doing one repo at a time.
(Still in progress, started today)

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+endif
+conf.set('ENABLE_IPV6', have_ipv6)
+
+################################################################################
+# Additionnal flags
+
+rootInclude = include_directories('.')
+
+pkgdatadir = join_paths(get_option('datadir'), meson.project_name())
+libexecdir = join_paths(get_option('libdir'),  meson.project_name())
```
Comment (leigh123linux) [inline meson.build]: Please change ('libdir') to ('libexecdir')

**Comment by Salamandar** [comment]
Libexecdir does not exist... That's why I'm creating this variable.

File: meson.build (hunk 1 above)
Comment (Salamandar) [reply meson.build]: Libexecdir does not exist... That's why I'm creating this variable.

Comment (leigh123linux) [reply meson.build]: You set it at build time

```
meson --libexecdir=/usr/libexec  whatever
```

https://github.com/mesonbuild/meson/commit/11f63105d003e222576a4e23ab106af2fc5a7741

Comment (leigh123linux) [reply meson.build]: You will also need to add --libexecdir=whatever to debian rules file to suit.

Comment (leigh123linux) [reply meson.build]: Perhaps you should look at

https://github.com/linuxmint/nemo/commit/eb342fd1569ad95b48bd6e2fa0555f1d1726fdc9
https://github.com/linuxmint/nemo/commit/c84980ab8583dfef8599c5c6aa5889e9fb3e78ae

Comment (Salamandar) [reply meson.build]: That's a feature coming in Meson 0.45.

**Comment by clefebvre** [comment]
One more meson project! :)

Great work.

**Comment by Salamandar** [comment]
Yeah o/

**Pull request merged**